helix-rest/src/main/java/org/apache/helix/rest/server/ServerContext.java

```diff
@@ -35,10 +36,12 @@
 import org.apache.helix.manager.zk.ZkClient;
 import org.apache.helix.manager.zk.client.HelixZkClient;
 import org.apache.helix.manager.zk.client.SharedZkClientFactory;
+import org.apache.helix.rest.server.resources.helix.PropertyStoreAccessor;
 import org.apache.helix.task.TaskDriver;
 import org.apache.helix.tools.ClusterSetup;
 
 public class ServerContext {
+  private static ZkBaseDataAccessor<ZNRecord> PROPERTY_STORE_ACCESSOR = null;
```

Comment:
Isn't ServerContext itself kind of a static concept in Helix REST? Any reason you're making this a static var?

Reply:
No specific reason, it also works with simple `private ZkBaseDataAccessor<ZNRecord> propertyStoreAccessor`